src/twister2/device/device_abstract.py

```diff
@@ -57,5 +60,19 @@ def flash_and_run(self, timeout: float = 60.0) -> None:
     def iter_stdout(self) -> Generator[str, None, None]:
         """Iterate stdout from a device."""
 
+    @abc.abstractmethod
+    def initialize_log_files(self, build_dir: str | Path):
+        """
+        Initialize file to store logs.
+
+        :param build_dir: path to directory with built application
+        """
+
+    def write_handler_log(self, stream: str) -> None:
+        if stream:
+            self.handler_log_file.handle(stream)  # type: ignore[union-attr]
+        else:
+            logger.debug('Nothing to write into %s', self.handler_log_file.filename)  # type: ignore[union-attr]
```

Comment:
do we really need this debug here? If DUT sends many empty lines, than we only log many debugs on the console, without updating the handle log file.

Reply:
and we do not need this method .. IMO better just write logs directly, using `self.handler_log_file.handle(line)`

Reply:
Done